src/cdogs/hud/player_hud.c

```diff
@@ -660,7 +660,7 @@ static void DrawPlayerObjectiveCompass(
 	{
 		return;
 	}
-	Rect2i r = Rect2iNew(svec2i_zero(), hud->device->cachedConfig.Res);
+	Rect2i r = Rect2iNew(svec2i(hud->device->cachedConfig.Res.x / 4, hud->device->cachedConfig.Res.y / 4), svec2i(hud->device->cachedConfig.Res.x / 2, hud->device->cachedConfig.Res.y / 2));
```

Comment:
please add a small comment explaining what this does (the /4, /2 stuff, which draws the arrows half way from the edge, centered)